lib/common/cmn_files.ml

```diff
@@ -0,0 +1,14 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+exception Cmn_write_error
+
+let write root name contents =
```

Comment:
Shouldn't this use the function you created in afin?

https://github.com/afiniate/afin/pull/2/files